src/pages/MyTransfers/TableFilters.styled.js

```diff
@@ -71,4 +71,10 @@ export const DateRangeButton = styled(Button)({
 
 export const DateRangeFilterIcon = styled(DateRangeIcon)({
   color: '#86c232',
+});
+
+export const FilterResetButton = styled(Button)({
+  color: '#fff',
+  height: '40px',
```

Comment:
I see that you mostly use px in a markup, and it is maybe not a problem here as it is a small button, but I would suggest using relative units instead like rem, so it would look similar on different scales.